### ISSUE TITLE
failed to mount as non-root user

### ISSUE THREAD

**Issue by multiSnow** [relan/exfat#133]
run:
```
$>> mkdir mnt
$>> mount.exfat-fuse -o noatime /dev/sdX1 ./mnt/
FUSE exfat 1.3.0
fusermount: unknown option 'user=<current user name>'
```
and exit with code 1 with no volume mounted.

If try again
```
$>> mount.exfat-fuse -o noatime /dev/sdX1 ./mnt/
FUSE exfat 1.3.0
WARN: volume was not unmounted cleanly.
fusermount: unknown option 'user=<current user name>'
```
but the volume should never mounted before.

Use `owner` option in `/etc/fstab` shows the same error.

Call `mount.exfat-fuse` as root works correctly.

**Comment by stratact** [comment]
Same issue here when using FreeBSD 12.1-STABLE.

**Comment by andersbo87** [comment]
Same issue on FreeBSD 12.1

**Comment by Alexco500** [comment]
Still an issue. It seems that the user option is currently not supported on FreeBSD. Changing main.c in line 502 from
`#if defined(__linux__) || defined(__FreeBSD__)`
to
`#if defined(__linux__) `

which should disable this.

**Comment by relan** [comment]
Fixed in 11e4f032d1782a606e3b004c89990a72cadf91bf.

**Comment by relan** [comment]
Fixed in v1.4.0.
